client/view/showStats.js

```diff
@@ -0,0 +1,141 @@
+import wikipediaLinks from "../wikipedia-links.json";
+
+const WIKIPEDIA_URL = 'https://en.wikipedia.org/wiki/'
+
+export function updateGlobalCoverageBar(data) {
+  const element = document.getElementById('global-coverage-bar');
+  element.innerHTML = '';
+  data.forEach((item) => {
+    const alpha = 1 - 1/(item.coverage);
+    const itemElem = document.createElement('li');
+    itemElem.classList.add('BrowsersStat__globalCoverageBarItem');
+    itemElem.setAttribute('style', `
+    --p: ${item.coverage};
```

Comment:
Setting CSS as a string has 2 problems:

1. JS minifier will not remove indent
2. Browser will run CSS parser

```js
itemElem.style.setProperty('--p', item.coverage)
```

Will be better. Here is a reference https://github.com/evilmartians/oklch-picker/blob/main/view/sample/index.ts#L15-L16